### DISCUSSION TITLE
Parsing Json Columns

### DISCUSSION THREAD

**Discussion by bryan-alvaradofusd** [grafana/grafana-infinity-datasource#98] [Q&A] [answered]
Hi, how would I go about parse a column that contains a json object? Here is an example:
Current:
id: 1 
State: 2
json: {'FName': 'John', 'LName': 'Smith'}

What I'm looking for:
id: 1 
State: 2
'FName': 'John'
'LName': 'Smith'

**Comment by yesoreyeram** [comment, marked as answer]
Hope this will give you some idea

![image](https://user-images.githubusercontent.com/153843/124772494-fd3ea780-df33-11eb-8302-aa1702ae979d.png)
![image](https://user-images.githubusercontent.com/153843/124772590-13e4fe80-df34-11eb-8655-cd1d3b1e76ed.png)
